src/keylessBackup/KeylessBackupProgress.test.tsx

```diff
@@ -0,0 +1,82 @@
+import { createMockStore, getMockStackScreenProps } from 'test/utils'
+import { KeylessBackupFlow, KeylessBackupStatus } from 'src/keylessBackup/types'
+import { Screens } from 'src/navigator/Screens'
+import { fireEvent, render, waitFor } from '@testing-library/react-native'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import * as React from 'react'
+import KeylessBackupProgress from 'src/keylessBackup/KeylessBackupProgress'
+import { navigate, navigateHome, ensurePincode } from 'src/navigator/NavigationService'
+import { Provider } from 'react-redux'
+import { mocked } from 'ts-jest/utils'
+import { KeylessBackupEvents } from 'src/analytics/Events'
+
+jest.mock('src/navigator/NavigationService')
+jest.mock('src/analytics/ValoraAnalytics')
+
+function createStore(keylessBackupStatus: KeylessBackupStatus) {
+  return createMockStore({
+    keylessBackup: {
+      backupStatus: keylessBackupStatus,
+    },
+  })
+}
+
+function getProps() {
+  return getMockStackScreenProps(Screens.KeylessBackupProgress, {
+    keylessBackupFlow: KeylessBackupFlow.Setup,
+  })
+}
+
+describe('KeylessBackupProgress', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+  it('navigates to home on success', async () => {
```

Comment:
can we add a test for the in progress state and assert the right icon is rendered?